enos/ansible/group_vars/all.yml

```diff
@@ -51,3 +51,8 @@ patches:
     src: mariadb_bootstrap.yml
     dst: kolla/ansible/roles/mariadb/tasks/bootstrap.yml
     enabled: "no"
+
+# docker images
+grafana_docker_image: grafana/grafana:5.1.3
+influxdb_docker_image: influxdb:1.5.3 
+cadvisor_docker_image: google/cadvisor:v0.30.0
```

Comment:
any reason for getting this defined here and in `roles/cadvisor/defaults/main.yml`?

Reply:
- Having those in `roles/*/defaults/main.yml` allows to have self-content roles
- Having those in `globals.yml` allow to centralize the current versions used in Enos.

Reply:
> Having those in roles/*/defaults/main.yml allows to have self-content roles

Oh, with a view to reuse roles in other projects I guess. Got it.